File: CS/WebSite/DevExpress/ASPxSpellCheckerForms/SpellCheckForm.ascx.cs
using System;
using System.Data;
using System.Configuration;
using System.Collections;
using System.Web;
using System.Web.Security;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using System.Web.UI.HtmlControls;
using DevExpress.Web.ASPxSpellChecker;
using DevExpress.Web.ASPxEditors;
using DevExpress.Web.ASPxSpellChecker.Localization;

public partial class SpellCheckForm : SpellCheckerUserControl {
    protected void Page_Load(object sender, EventArgs e) {
        PrepareCheckedDiv();
        PrepareAddToDictionaryButton();
		Localize();
    }
    protected override ASPxEditBase[] GetChildDxEdits() {
        return new ASPxEditBase[] { txtChangeTo, SCSuggestionsListBox };
    }

    protected override ASPxButton[] GetChildDxButtons() {
        return new ASPxButton[] {
            btnAddToDictionary,
            btnChange,
            btnChangeAll,
            btnClose,
            btnIgnore,
            btnIgnoreAll,
            btnOptions
        };
    }

    protected void PrepareAddToDictionaryButton() {
        btnAddToDictionary.Enabled = SpellChecker.GetCustomDictionary() != null;
    }
    protected void PrepareCheckedDiv() {
        CheckedTextContainerStyle style = SpellChecker.GetCheckedTextContainerStyle();
        SCCheckedDiv.Attributes.Add("class", style.CssClass);
        CssStyleCollection styles = style.GetStyleAttributes(Page);
        foreach (string key in styles.Keys)
            SCCheckedDiv.Style.Add(key, styles[key]);
    }
	protected void Localize() {
		btnIgnore.Text = ASPxSpellCheckerLocalizer.GetString(ASPxSpellCheckerStringId.IgnoreOnceButton);
		btnIgnoreAll.Text = ASPxSpellCheckerLocalizer.GetString(ASPxSpellCheckerStringId.IgnoreAllButton);
		btnAddToDictionary.Text = ASPxSpellCheckerLocalizer.GetString(ASPxSpellCheckerStringId.AddToDictionaryButton);
		btnChange.Text = ASPxSpellCheckerLocalizer.GetString(ASPxSpellCheckerStringId.ChangeButton);
		btnChangeAll.Text = ASPxSpellCheckerLocalizer.GetString(ASPxSpellCheckerStringId.ChangeAllButton);
		btnOptions.Text = ASPxSpellCheckerLocalizer.GetString(ASPxSpellCheckerStringId.OptionsButton);
		btnClose.Text = ASPxSpellCheckerLocalizer.GetString(ASPxSpellCheckerStringId.CloseButton);
	}
}

File: VB/WebSite/DevExpress/ASPxSpellCheckerForms/SpellCheckForm.ascx.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Data
Imports System.Configuration
Imports System.Collections
Imports System.Web
Imports System.Web.Security
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports System.Web.UI.WebControls.WebParts
Imports System.Web.UI.HtmlControls
Imports DevExpress.Web.ASPxSpellChecker
Imports DevExpress.Web.ASPxEditors
Imports DevExpress.Web.ASPxSpellChecker.Localization

Public Partial Class SpellCheckForm
	Inherits SpellCheckerUserControl
	Protected Sub Page_Load(ByVal sender As Object, ByVal e As EventArgs)
		PrepareCheckedDiv()
		PrepareAddToDictionaryButton()
		Localize()
	End Sub
	Protected Overrides Function GetChildDxEdits() As ASPxEditBase()
		Return New ASPxEditBase() { txtChangeTo, SCSuggestionsListBox }
	End Function

	Protected Overrides Function GetChildDxButtons() As ASPxButton()
		Return New ASPxButton() { btnAddToDictionary, btnChange, btnChangeAll, btnClose, btnIgnore, btnIgnoreAll, btnOptions }
	End Function

	Protected Sub PrepareAddToDictionaryButton()
		btnAddToDictionary.Enabled = Not SpellChecker.GetCustomDictionary() Is Nothing
	End Sub
	Protected Sub PrepareCheckedDiv()
		Dim style As CheckedTextContainerStyle = SpellChecker.GetCheckedTextContainerStyle()
		SCCheckedDiv.Attributes.Add("class", style.CssClass)
		Dim styles As CssStyleCollection = style.GetStyleAttributes(Page)
		For Each key As String In styles.Keys
			SCCheckedDiv.Style.Add(key, styles(key))
		Next key
	End Sub
	Protected Sub Localize()
		btnIgnore.Text = ASPxSpellCheckerLocalizer.GetString(ASPxSpellCheckerStringId.IgnoreOnceButton)
		btnIgnoreAll.Text = ASPxSpellCheckerLocalizer.GetString(ASPxSpellCheckerStringId.IgnoreAllButton)
		btnAddToDictionary.Text = ASPxSpellCheckerLocalizer.GetString(ASPxSpellCheckerStringId.AddToDictionaryButton)
		btnChange.Text = ASPxSpellCheckerLocalizer.GetString(ASPxSpellCheckerStringId.ChangeButton)
		btnChangeAll.Text = ASPxSpellCheckerLocalizer.GetString(ASPxSpellCheckerStringId.ChangeAllButton)
		btnOptions.Text = ASPxSpellCheckerLocalizer.GetString(ASPxSpellCheckerStringId.OptionsButton)
		btnClose.Text = ASPxSpellCheckerLocalizer.GetString(ASPxSpellCheckerStringId.CloseButton)
	End Sub
End Class